.eslintrc.js

```diff
@@ -1,10 +1,28 @@
-/**
- * These rules enforce the Hack Reactor Style Guide
- *
- * Visit this repo for more information:
- *   https://github.com/reactorcore/eslint-config-hackreactor
- */
-
 module.exports = {
-  extends: './node_modules/eslint-config-hackreactor/index.js'
-};
\ No newline at end of file
+  env: {
+    browser: true,
+    commonjs: true,
+    es6: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+  },
```

Comment:
I added 'no-console': 'off' to rules so I could use console.log without getting yelled at.

Reply:
Awesome! Thank you!